src/style.css

```diff
@@ -0,0 +1,150 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --botton-color: #00acdc;
+  --accent-color-text: #060b35;
+}
+
+.block-cards {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  background: #fff;
+  width: 200px;
+  font-family: "Roboto", sans-serif;
```

Comment:
move to `html`

Reply:
![image](https://user-images.githubusercontent.com/101557993/179243920-5bbcf3ef-ec45-4bc0-803d-c1e4e1fd6218.png)